draft-ietf-openpgp-pqc.md

```diff
@@ -321,9 +321,8 @@ Furthermore, multiple OpenPGP signatures may be combined on the application laye
 These latter two cases realize non-composite combinations of signatures.
 {{multiple-signatures}} specifies how implementations should handle the verification of such combinations of signatures.
 
-Furthermore, the OpenPGP protocol also allows for parallel encryption to different keys held by the same recipient.
-Accordingly, if the sender makes use of this feature and sends an encrypted message with multiple PKESK packages for different encryption keys held by the same recipient, a non-composite multi-algorithm public-key encryption is realized where the recipient has to decrypt only one of the PKESK packages in order to decrypt the message.
-See {{no-pq-t-parallel-encryption}} for restrictions on parallel encryption mandated by this specification.
+Furthermore, the OpenPGP protocol also allows for parallel encryption to different keys.
+Accordingly, when a message is encrypted using multiple PKESK packages for different encryption keys, a non-composite, potentially multi-algorithm, public-key encryption is realized where the communication party has to decrypt only one of the PKESK packages in order to decrypt the message.
```

Comment:
what is a "PKESK package"?  does this mean "packet"?  i recognize it's not a part of the text edited here, but it could be cleaned up while we're touching this line.

again, i think "communication party" is too generic.  we're talking about the "consuming" or "evaluating" implementation, not the generating implementation.  in this case, we're just talking about decryption, so we could just say "the decrypting party".